test/cmd/newapp.sh

```diff
@@ -9,10 +9,18 @@ source "${OS_ROOT}/hack/util.sh"
 source "${OS_ROOT}/hack/cmd_util.sh"
 os::log::install_errexit
 
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc delete all,templates --all
+  exit 0
+) &>/dev/null
```

Comment:
I could document that during development one could run `test/cmd/*.sh` files to test against a running server.

However, this also means that running a test might accidentally delete resources you had in the server.

Could we protect this clean up somehow? Say, only delete resources if an environment variable `OPENSHIFT_TEST_CMD_FORCE_CLEANUP` is set.
Then, by default, running a test would not delete resources in whatever namespace/project you happen to be.